waku/v2/protocol/waku_rln_relay/waku_rln_relay_types.nim

```diff
@@ -90,7 +90,7 @@ type MessageValidationResult* {.pure.} = enum
 # inputs of the membership contract constructor
 # TODO may be able to make these constants private and put them inside the waku_rln_relay_utils
 const
-  MEMBERSHIP_FEE* = 5.u256
+  MEMBERSHIP_FEE* = 1000000000000000.u256
   #  the current implementation of the rln lib only supports a circuit for Merkle tree with depth 32
```

Comment:
```suggestion
  #  the current implementation of the rln lib supports a circuit for Merkle tree with depth 20
```

Reply:
Thanks, fixed in d5c60f2aa16db9cba15df40dd60171f1318531ef